blog/2019-12-18-debezium-1-0-0-final-released.adoc

```diff
@@ -0,0 +1,140 @@
+just by means of configuration.
+
+But there's many other use cases of CDC which go beyond just moving data from A to B.
+When adding stream processing into the picture, e.g. via Kafka Streams or Apache Flink,
+CDC enables you to run https://github.com/debezium/debezium-examples/tree/master/kstreams-live-update[time-windowed streaming queries], continuously updated as your data changes
+("what's the aggregated order revenue per category within the last hour").
+You can use CDC to link:/blog/2019/10/01/audit-logs-with-change-data-capture-and-stream-processing/[build audit logs] of your data,
+telling who did change which data items at which time.
+Or update denormalized views of your data, for the sake of efficient data retrieval, adhering to the CQRS pattern.
+
+Finally, CDC also can play a vital role in microservices architectures;
+exchanging data between services and keeping local views of data owned by other services achieves a higher independence, without having to rely on synchronous API calls.
+One particularly interesting approach in this context is the link:/blog/2019/02/19/reliable-microservices-data-exchange-with-the-outbox-pattern/[outbox pattern],
+which is link:/documentation/reference/1.0/configuration/outbox-event-router.html[supported well] by Debezium.
+In case you don't start on the green field (who ever does?),
+CDC can be used to implement the strangler pattern for moving from a monolithic design to microservices.
+
+You can learn more about change data capture use cases with Debezium and Apache Kafka in https://speakerdeck.com/gunnarmorling/practical-change-data-streaming-use-cases-with-apache-kafka-and-debezium-qcon-san-francisco-2019[this presentation] from QCon San Francisco.
+
+But you don't have to take our word for it:
+you can find lots of blog posts, conference talks and examples by folks using Debezium in production in our compilation of link:/documentation/online-resources/[resources].
+If you'd like to get a glimpse of who else already is using Debezium,
+see our rapidly growing link:/community/users/[list of reference users]
+(or send us a https://github.com/debezium/debezium.github.io/blob/develop/community/users.asciidoc[pull request] to get your name added if your organization already is running Debezium in production).
+
+== Debezium 1.0
+
+Now, let's talk a little bit about the contents of the 1.0 release.
+
+This version continues the efforts we've begun in link:/blog/2019/10/02/debezium-0-10-0-final-released/[0.10] to make sure the emitted event structures and configuration options of the connectors are correct and consistent.
+While we've always been very careful to ensure a smooth upgrading experience, you can expect even more stability in this regard going forward after the 1.0 release.
+
+We've expanded the test coverage of databases (Postgres 12, SQL Server 2019),
+build against the latest version of Apache Kafka (2.4.0) and upgraded our container images to OpenJDK 11.
```

Comment:
The present tense of "build" conflicts with the past tense of "expanded" and "upgraded". But using "built" doesn't really work either. So how about 

We've expanded the test coverage of databases (Postgres 12, SQL Server 2019),
upgraded our container images to OpenJDK 11 and we build against the latest version of Apache Kafka (2.4.0).

which sounds better to me, even if not perfect.

Reply:
Thx, changed it along these lines.